Doesn't build on Mono 3.8.0

When building on Mono 3.8.0, the following error occurs :

```
/Users/Stephen/Dropbox/Code/omnisharp-atom/OmniSharpServer/OmniSharp.sln (default targets) ->
(Build target) ->
/Users/Stephen/Dropbox/Code/omnisharp-atom/OmniSharpServer/OmniSharp/OmniSharp.csproj (default targets) ->
/Library/Frameworks/Mono.framework/Versions/3.8.0/lib/mono/4.5/Microsoft.CSharp.targets (CoreCompile target) ->

    Bootstrapper.cs(41,37): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToStartOfPipeline(System.Func<Nancy.NancyContext,Nancy.Response>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToStartOfPipeline(Nancy.PipelineItem<System.Func<Nancy.NancyContext,Nancy.Response>>, bool)'
    Bootstrapper.cs(42,36): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToEndOfPipeline(System.Action<Nancy.NancyContext>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToEndOfPipeline(Nancy.PipelineItem<System.Action<Nancy.NancyContext>>, bool)'
    Bootstrapper.cs(46,41): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToStartOfPipeline(System.Func<Nancy.NancyContext,Nancy.Response>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task<Nancy.Response>>,System.Func<Nancy.NancyContext,Nancy.Response>>.AddItemToStartOfPipeline(Nancy.PipelineItem<System.Func<Nancy.NancyContext,Nancy.Response>>, bool)'
    Bootstrapper.cs(47,40): error CS0121: The call is ambiguous between the following methods or properties: `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToStartOfPipeline(System.Action<Nancy.NancyContext>)' and `Nancy.AsyncNamedPipelineBase<System.Func<Nancy.NancyContext,System.Threading.CancellationToken,System.Threading.Tasks.Task>,System.Action<Nancy.NancyContext>>.AddItemToStartOfPipeline(Nancy.PipelineItem<System.Action<Nancy.NancyContext>>, bool)'
```

--- a/OmniSharp/Bootstrapper.cs
+++ b/OmniSharp/Bootstrapper.cs
@@ -37,14 +37,14 @@ namespace OmniSharp
             {
                 HelpService.AsyncInitialize();
             }
-            
-            pipelines.BeforeRequest.AddItemToStartOfPipeline(StopWatchStart, false);
-            pipelines.AfterRequest.AddItemToEndOfPipeline(StopWatchStop, false);
+
+            pipelines.BeforeRequest += (ctx) => StopWatchStart (ctx);
+            pipelines.AfterRequest += (ctx) => StopWatchStop (ctx);
 
             if (_logger.Verbosity == Verbosity.Verbose)
             {
-                pipelines.BeforeRequest.AddItemToStartOfPipeline(LogRequest, false);
-                pipelines.AfterRequest.AddItemToStartOfPipeline(LogResponse, false);
+                pipelines.BeforeRequest += (ctx) => LogRequest (ctx);
+                pipelines.AfterRequest += (ctx) => LogResponse (ctx);
             }
 
             pipelines.OnError.AddItemToEndOfPipeline((ctx, ex) =>

